cmd/auth-rest/startcmd/start.go

```diff
@@ -86,13 +122,31 @@ const (
 var logger = log.New("auth-rest")
 
 type authRestParameters struct {
-	hostURL           string
-	tlsSystemCertPool bool
-	tlsCACerts        []string
-	logLevel          string
-	databaseType      string
-	databaseURL       string
-	databasePrefix    string
+	hostURL        string
+	logLevel       string
+	databaseType   string
+	databaseURL    string
+	databasePrefix string
+	tlsParams      *tlsParams
+	oidcParams     *oidcParams
+}
+
+type tlsParams struct {
+	useSystemCertPool bool
+	caCerts           []string
+	serveCertPath     string
+	serveKeyPath      string
```

Comment:
we fall into the same config issue we had in Fabric SDK Go and Fabric Extensions.. using path vs embedding certs.. do we wanna only support path for now? or should we also support embedded certs/keys?

cc @troyronda

Reply:
I guess `srv.ListenAndServeTLS()` takes only paths for now? It should be ok then.

Reply:
@Baha-sk would be helpful if you could elaborate on the issue?

Reply:
In other projects, we added the ability to embed a full cert content into the configs to avoid reading it from a path (in case the code doesn't have read access to the FS), but I guess these fields here are only used for TLS configs of a server which expects path only. There's no point in adding the embedded certs in the configs in this case.

My original comment can be ignored.